src/VolumeClaimCommands.cpp

```diff
@@ -88,9 +88,15 @@ crow::response listVolumeClaims(PersistentStore& store, const crow::request& req
 		} 
 
 		// Add volume status from K8s (Bound, Pending...)
+		if((volumeStatus.IsObject() && volumeStatus.HasMember("status"))
+		   && volumeStatus["status"].IsObject() && volumeStatus["status"].HasMember("phase")){
 		volumeData.AddMember("status", volumeStatus["status"]["phase"], alloc);
 		volumeResult.AddMember("metadata", volumeData, alloc);
```

Comment:
This and the next line should not be inside the conditional block, but entirely after it (that is, after the `else` block) so that they are always executed.